src/include/wiredtiger.in

```diff
@@ -64,8 +64,7 @@ extern "C" {
 struct __wt_collator;	    typedef struct __wt_collator WT_COLLATOR;
 struct __wt_compressor;	    typedef struct __wt_compressor WT_COMPRESSOR;
 struct __wt_config_item;    typedef struct __wt_config_item WT_CONFIG_ITEM;
-struct __wt_config_parser;
-	typedef struct __wt_config_parser WT_CONFIG_PARSER;
+struct __wt_config_parser;  typedef struct __wt_config_parser WT_CONFIG_PARSER;
```

Comment:
What changed here in the whitespace presumably that resulted in this? Is it intentional?

Reply:
This line looked different than the others for no reason, but it doesn't need to be fixed now.